livekit-plugins/livekit-plugins-aws/livekit/plugins/aws/tts.py

```diff
@@ -0,0 +1,208 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import asyncio
+from dataclasses import dataclass
+from typing import Any, Callable
+
+import aiohttp
+from aiobotocore.session import AioSession, get_session
+from livekit import rtc
+from livekit.agents import (
+    DEFAULT_API_CONNECT_OPTIONS,
+    APIConnectionError,
+    APIConnectOptions,
+    APIStatusError,
+    APITimeoutError,
+    tts,
+    utils,
+)
+
+from ._utils import (
+    TTS_LANGUAGE,
+    TTS_OUTPUT_FORMAT,
+    TTS_SPEECH_ENGINE,
+    _get_aws_credentials,
+)
+
+TTS_NUM_CHANNELS: int = 1
+
+
+@dataclass
+class _TTSOptions:
+    # https://docs.aws.amazon.com/polly/latest/dg/API_SynthesizeSpeech.html
+    voice: str | None
+    output_format: TTS_OUTPUT_FORMAT
+    speech_engine: TTS_SPEECH_ENGINE
+    speech_region: str | None
+    sample_rate: int
+    language: TTS_LANGUAGE | None
+
+
+class TTS(tts.TTS):
+    def __init__(
+        self,
+        *,
+        voice: str | None = "Ruth",
+        language: TTS_LANGUAGE | None = None,
```

Comment:
```suggestion
        language: TTS_LANGUAGE | None = None,
```
We should always allow a str too here, we can't guarantee we will update the languages quickly